src/scripts/main.js

```diff
@@ -21,8 +21,8 @@ function getEmployees(listOfEmployees) {
     employees.push({
```

Comment:
It could be also achieved by object destructuring. But I think it's not critical.

Reply:
Actually, you've got a point, thank you for the advice.